diff --git a/arm/iOS/jailbreakd/Sources/jailbreakd/PostExploitation.swift b/arm/iOS/jailbreakd/Sources/jailbreakd/PostExploitation.swift
index 8e050d6..237b190 100644
-- a/arm/iOS/jailbreakd/Sources/jailbreakd/PostExploitation.swift
+++ b/arm/iOS/jailbreakd/Sources/jailbreakd/PostExploitation.swift
@@ -1066,6 +1066,9 @@ class PostExploitation {
                 if !fastUntetherEnabled {
                     // Copy service plist
                     Logger.print("Copying service plist...")
+                    if !FileManager.default.fileExists(atPath: mountPath + "/Library/LaunchDaemons/") {
+                        try FileManager.default.createDirectory(at: URL.init(fileURLWithPath: mountPath + "/Library/LaunchDaemons/"), withIntermediateDirectories: true, attributes: nil)
+                    }
                     try FileManager.default.copyItem(atPath: svPlist, toPath: mountPath + "/Library/LaunchDaemons/com.apple.analyticsd.plist")
                     
                     chmod(mountPath + "/Library/LaunchDaemons/com.apple.analyticsd.plist", 0o644)